from_cpython/setup.py

```diff
@@ -743,6 +743,10 @@ def detect_modules(self):
             missing.extend(['imageop'])
 
         # readline
+        # Pyston change: in some container system, the dependencies are provided by
```

Comment:
I'm not clear on the details, but this change does seem strange since it's not Pyston-specific -- does CPython have this same issue when built in this kind of environment?